object_based.py

```diff
@@ -47,17 +47,17 @@ def count_cooccurrence(dataloader, args):
         for j in range(i+1, len(all_categories)):
             counts["{0}-{1}".format(i, j)] = 0
 
-    groupings_size = [[] for i in range(len(GROUPINGS_TO_NAMES))]
-    groupings_dist = [[] for i in range(len(GROUPINGS_TO_NAMES))]
+    groupings_size = [[] for i in range(len(DEFAULT_GROUPINGS_TO_NAMES))]
```

Comment:
instead of DEFAULT_GROUPINGS_TO_NAMES throughout these files, it should say dataloader.dataset.supercategories_to_names